contracts/ClaimAndStake.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Multicall.sol";
+import "@uma/core/contracts/merkle-distributor/implementation/MerkleDistributorInterface.sol";
+
+interface AcceleratingDistributorInterface {
+    function stakeFor(
+        address beneficiary,
+        address stakedToken,
+        uint256 amount
+    ) external;
+}
+
+interface AcrossMerkleDistributorInterface is MerkleDistributorInterface {
+    function claimFor(MerkleDistributorInterface.Claim memory _claim) external;
+}
```

Comment:
nit: I know we do this elsewhere, but is there any reason not to just import these contracts directly? I only suggest it since it minimizes the chance of error (like a typo in the minimal interface that doesn't match the real contract).

Reply:
For contract size purposes but I guess that's not really a concern since this contract is so small